### PR TITLE
feat(ci) Enable CI on project with 2 example of specs

### PR DESCRIPTION
**Pull request by Soulou**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+.DEFAULT := test
+
+test: BASH_COMMAND := test/run
+test: docker
+
+docker:
+	@echo "Running tests in Docker using $(IMAGE)"
+	@docker pull $(IMAGE)
+	@docker run -v $(PWD):/buildpack:ro --rm --user appsdeck -it -e "GITLAB_TOKEN=$(GITLAB_TOKEN)" -e "GITHUB_TOKEN=$(GITHUB_TOKEN)" -e "STACK=$(STACK)" $(IMAGE) bash -c "cd /buildpack; $(BASH_COMMAND)"
```
Comment (Frzk) [inline Makefile]: I think we can remove the `@docker pull $(IMAGE)` and replace it with `@docker run --pull always ...`

**Review by Frzk — APPROVED** [review]
Nice work <3

File: Makefile (hunk 1 above)
Comment (Soulou) [reply Makefile]: Done

Comment (Frzk) [reply Makefile]: Ship it 🤩

**Pull request merged**